<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BAH5Q3C9J3" failures="2" tests="10" name="RetryMulitple" time="0.044" errors="0" timestamp="2023-03-16T10:02:09 EDT">
  <testcase classname="TestNGSamples2.TestRetry001" name="Test1" time="0.008">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test1" time="0.0">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test1" time="0.001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test1" time="0.0">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGSamples2.TestRetry001.Test1(TestRetry001.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test2" time="0.001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test2" time="0.0">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test2" time="0.0">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test2" time="0.0">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="TestNGSamples2.TestRetry001" name="Test2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGSamples2.TestRetry001.Test2(TestRetry001.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- RetryMulitple -->
